src/vcpkg/binarycaching.cpp

```diff
@@ -31,42 +31,44 @@ namespace
                                  (msg::count, msg::vendor),
                                  "",
                                  "Attempting to fetch {count} package(s) from {vendor}");
-    DECLARE_AND_REGISTER_MESSAGE(RestoredPackagesFromVendor,
-                                 (msg::count, msg::elapsed, msg::vendor),
-                                 "",
-                                 "Restored {count} package(s) from {vendor} in {elapsed}");
-    DECLARE_AND_REGISTER_MESSAGE(UploadedPackagesToVendor,
-                                 (msg::count, msg::elapsed, msg::vendor),
-                                 "",
-                                 "Uploaded {count} package(s) to {vendor} in {elapsed}");
-    DECLARE_AND_REGISTER_MESSAGE(UnknownBinaryProviderType,
-                                 (),
-                                 "",
-                                 "unknown binary provider type: valid providers are 'clear', 'default', 'nuget', "
-                                 "'nugetconfig', 'interactive', and 'files'");
-    DECLARE_AND_REGISTER_MESSAGE(InvalidArgumentRequiresPrefix,
-                                 (msg::binary_source),
+    DECLARE_AND_REGISTER_MESSAGE(
+        AuthenticationMayRequireManualAction,
+        (msg::vendor),
+        "",
+        "One or more {vendor} credential providers requested manual action. Add the binary source "
+        "'interactive' to allow interactivity.");
+    DECLARE_AND_REGISTER_MESSAGE(CompressFolderFailed, (msg::path), "", "Failed to compress folder '{path}':");
+    DECLARE_AND_REGISTER_MESSAGE(
+        DefaultPathToBinaries,
+        (msg::path),
+        "",
+        "Based on your system settings, the default path to store binaries is \n '{path}'. This consults "
+        "%LOCALAPPDATA%/%APPDATA% on Windows and $XDG_CACHE_HOME or $HOME on other platforms.");
```

Comment:
Don't use escape sequences in these messages.
If you must have a new line then split the message in two and use `LocalizedString::appendnl()`. In my opinion, you don't need the new line before the `{path}`.

Reply:
`appendnl()` was removed by @BillyONeal. Use `.append_raw('\n')` instead.

Reply:
https://github.com/microsoft/vcpkg-tool/pull/547

Reply:
See also https://github.com/microsoft/vcpkg-tool/pull/553 which enforces that localized messages don't *end* with `\n` but allows them in the middle.

Reply:
I think it's fine then.